server/src/main/java/io/spine/server/bus/UnicastBus.java

```diff
@@ -48,7 +48,7 @@ protected D getDispatcher(E envelope) {
 
     private static IllegalStateException noDispatcherFound(MessageEnvelope envelope) {
         String id = Identifier.toString(envelope.id());
-        String msg = format("No dispatcher found for the command (class: %s id: %s).",
+        String msg = format("No dispatcher found for the command (class: `%s` id: `%s`).",
```

Comment:
I'd personally rather extract `format` string as a separate variable and inline all 3 parameters of the `format()` call:
```
String msg = format(messageFormat, envelope.messageClass(), id);
```